pkg/network/infraconfigurators/bridge.go

```diff
@@ -197,6 +193,17 @@ func (b *BridgePodNetworkConfigurator) createBridge() error {
 		return err
 	}
 
+	//replace podNic's mac with bridge's mac before linking podIface as master on bridge
```

Comment:
Well, I see this written just  below using code (which is not really complex). I do not see additional information which is beyond the code (e.g. reasons or some context to the outside).

If you still think it is important, I prefer to see comments on the function description.

Reply:
fixed